charts/producer-app/templates/pod.yaml

```diff
@@ -1,19 +1,19 @@
 {{- define "producer-app.podTemplate" -}}
 {{- $root := . -}}
 metadata:
-{{- if or .Values.awsRole .Values.annotations }}
```

Comment:
same comment as above

Reply:
> same comment as above

producer-app already has Values.annotations in deployment.yaml. 
but there are only Values.labels in job.yaml. Should I add it there?

Reply:
yes please